### PR TITLE
Pull requests api cleanup

### PR DESCRIPTION
**Pull request by bandjeo**
Responded to requested changes.

### CODE REVIEW

File: oss-cloud-backend/handler.js [hunk 1]
```diff
@@ -28,7 +28,7 @@ module.exports.addContributor = async (event, context, callback) => {
   // check if request is valid
   try {
     // TODO create generic function for checking validity of a body
-    var body = JSON.parse(event.body);
+    let body = JSON.parse(event.body);
     // check if keys exist
     body.username;
```
Comment (banecogic) [inline oss-cloud-backend/handler.js]: This line as well as following two can be removed or commented. There is one more occurrence of this type at line no. 105.

Comment (bandjeo) [reply oss-cloud-backend/handler.js]: Removed the three lines in the first occurrence. The second occurrence is already overwritten in another branch.

File: oss-cloud-backend/gitHubApiService.js [hunk 2]
```diff
@@ -45,17 +45,18 @@ module.exports.getRepoDetails = async repos => {
     return getRepo(repo.owner.login, repo.name);
   });
 
-  return await Promise.all(repoPromises);
-};
+  return Promise.all(repoPromises);
```
Comment (mjovanovic) [inline oss-cloud-backend/gitHubApiService.js]: The function shouldn't be marked async any more.

File: oss-cloud-backend/gitHubApiService.js [hunk 3]
```diff
@@ -78,22 +79,15 @@ searchUserPullRequests = async (username, repo) => {
...
-        per_page: 100
-      })
-      .then(({ data, headers, status }) => {
-        pullRequests = pullRequests.concat(data.items);
-        pullRequestNum = data.items.length;
-      });
+    await octokit.search.issuesAndPullRequests({
+      q: `repo:${repo.owner.login}/${repo.name}+author:${username}+is:pr`,
+      per_page: 100,
+      page: pullRequestNum
```
Comment (mjovanovic) [inline oss-cloud-backend/gitHubApiService.js]: This will be 100 for second request. Shouldn't it be 1? Or 2?

**Pull request merged**